lib/happy_hood/slack/client.rb

```diff
@@ -29,11 +29,23 @@ def self.daily_message
 
         def self.summarize_differences(differences)
           differences.reject { |d| d.valuation_difference.zero? }.map do |d|
-            "```"\
-            "(#{d.name}) - #{d.house_count} Happy Houses - (#{d.valuation_date})\n"\
-            "Yesterday: #{currency_format(d.yesterdays_valuation)}\n"\
-            "Today:     #{currency_format(d.todays_valuation)} (Difference: #{currency_format(d.valuation_difference)})\n"\
-            "```"
+            average_house_difference = if d.house_count > 1
+              avg_diff = d.valuation_difference / d.house_count
+
+              avg_diff_string = avg_diff.positive? ? "+#{currency_format(avg_diff)}" : "#{currency_format(avg_diff)}"
+
+              "(#{avg_diff_string} avg/house)"
+            end
```

Comment:
I almost feel like this should go into the `DailyDifferenceCalculator`, and this class should just focus on presentation details? What do you think?

Reply:
What are your thoughts on this now that the calculation was moved to the business logic layer?

The `average_house_difference` is now calculated in the `DailyDifferenceCalculator`, but there's still some minor presentation logic here.